front/app/containers/ProjectsShowPage/shared/header/ProjectInfo.tsx

```diff
@@ -248,8 +244,12 @@ const ProjectInfo = memo<Props>(({ projectId, className }) => {
                 </>
               )}
             </ProjectDescription>
+            {!isNilOrError(projectFiles) &&
```

Comment:
This was not within the QuillEditedContent, so it would not have worked with our new `disableTabbing` function. I moved it below the variable description (which can be shown/hidden) so it's always visible below the content, and so now will be tabbable like normal since it will always appear.